### ISSUE TITLE
iFrames are blocked by html configuration

### ISSUE THREAD

**Issue by lowstrife** [Tucsky/SignificantTrades#97]
Quick question\issue. 

I use iframes to combine multiple charting websites to see market data. 

![image](https://user-images.githubusercontent.com/65734844/82599358-5c433080-9b69-11ea-9b6d-3486c7035364.png)

And when I integrate aggr.trade to my iframe setup, it gets refused. Is this fixable? Or is it a security measure to prevent spoofing on other sites. Are there other workarounds that I can use to integrate all of these websites on a single page?

![image](https://user-images.githubusercontent.com/65734844/82599261-3158dc80-9b69-11ea-8b89-59f76130cc23.png)

**Comment by Tucsky** [comment]
SHould be fixed now. 
Sorry for the inconvenience

**Comment by lowstrife** [comment]
Fixed! Many thanks for the quick reply.

**Comment by lowstrife** [comment]
Further update: 

I managed to fix everything. I needed to add a cookie exception to aggr.trade in the web browser. This fixed not all of the exchanges connecting, and allowed my settings to be saved upon page refresh. Working perfectly now! Sorry for the spam of updates as I troubleshooted this.

![image](https://user-images.githubusercontent.com/65734844/82612015-198c5300-9b7f-11ea-92cb-76e85a89a3e9.png)
